GGProject/workflow/management/commands/sync_command.py

```diff
@@ -0,0 +1,50 @@
+import requests as req
+import ldap
+from django_auth_ldap.backend import LDAPBackend
+from django.core.management.base import BaseCommand, CommandError
+from GreeterGuru import settings
+
+
+class Command(BaseCommand):
+
+    help = "Synchronizes web database employees with active directory users"
+    
+    def handle(self, *args, **kwargs):
+
+        host = settings.SERVER_NAME		   # Server’s name/IP address
+        username = "internship\\Administrator" # Any user’s login name on the server
```

Comment:
Same here. Does it make more sense that we can access this string from settings.py?